### ISSUE TITLE
`start-env` directly from `comit-scripts` instead of using `create-comit-app`

### ISSUE THREAD

**Issue by da-kami** [comit-network/ambrosia#19]
Note: To be done after orderbook + swap is done - i.e. end to end from getting order of maker to finish the swap with the maker.

Depends on this to be fixed: https://github.com/comit-network/create-comit-app/issues/778
Also depends on releasing create-comit-app so all the fixes done on the way are in (e.g. Derivation path print fix, see: https://github.com/comit-network/create-comit-app/tree/wallet-descriptor-fix)

Basically do this in `package.json` of the app:

```json
{
  ...
  "scripts": {
    ...
    "start-env": "comit-scripts start-env"
  },
  ...
  "dev-dependencies": {
    "comit-scripts": "^0.9.0",
  }
}
```

Then just use `yarn start-env` in the app. No create-comit-app no more.

**Comment by da-kami** [comment]
@thomaseizinger thoughts?

**Comment by thomaseizinger** [comment]
That is pretty much how it is supposed to be used yes :)

`create-comit-app` should just be the command that you use to initially **create** your project. After that, you shouldn't need to touch it.

**Comment by da-kami** [comment]
does not have priority right now, closing. Reopen at later point if still deemed necessary.
